actors.md

```diff
@@ -530,6 +532,7 @@ type MinerInfo struct {
 | `ChangeWorker` | 13 |
 | `IsSlashed` |  14 |
 | `IsLate` | 15 |
+| `AddFault` | 16 |
```

Comment:
you once mentioned that we needed "recoveredFault" or something, do we still need that?